src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,87 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_FIELD = "supply";
+    private static final String BUY_FIELD = "buy";
+    private static final String RESULT_FIELD = "result";
+    private static final String LINES_SEPARATOR = " ";
+    private static final String WORDS_SEPARATOR = ",";
+    private static final int DATA_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private final String separator = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder report = countData(readFromFile(fromFileName));
+        writeToFile(report, toFileName);
+    }
+
+    private StringBuilder readFromFile(String fileName) {
```

Comment:
It is better to return String than StringBuilder
```suggestion
    private String readFromFile(String fileName) {
```